test/e2e/chainsaw/check-arbitary-objects/chainsaw-test.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: check-arbitrary-objects
+spec:
+  description: |
+    Tests if an instance with a key of type 'object' allows passing unstructured data
+  steps:
+  - name: install-rgd
+    try:
+    #description: Install the RGD that creates an Instance CRD
```

Comment:
Are these intentionally left as descriptions here, or were commented out in favor of the actual descriptions below and just not deleted?

Seems like it might be the latter?